vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -12,13 +12,13 @@ vignette: >
 
 ## Introduction
 
-`teal` version `0.16` introduced new argument in `teal::module` called `transformers`. 
+`teal` version `0.16` introduced new argument in `teal::module` called `transformators`. 
 This argument allows to pass a `list` of `teal_data_module` objects that are created using `teal_transform_module()` function.
```

Comment:
There is a statement in line 21 "When provided, ...". Move it here. Start with "This is an optional argument and if it is provided then ..."

Here we are discussing the argument and that statement is indeed an argument description

Reply:
Oh no - scratch that. We are describing the benefits of `transformators` argument - not the benefits of a function. That needs to be changed accordingly. Also probably should be moved upper so that we have: in new version we have added X. X is good because of .... In order to create X use foo_x(). foo_x() is ...